src/styles/styles.js

```diff
@@ -768,33 +768,32 @@ const styles = {
         borderRightWidth: 0,
     },
 
+    singleAvatarContainer: {
+        marginTop: -2,
```

Comment:
Can you explain what exactly this does?

Reply:
the previous implementation had something similar: https://github.com/Expensify/Expensify.cash/pull/1593/files#diff-8ab8ba45ab927f1430b8d1209d187e003a9cdfe476f809e4483ef64cdba253ebL786

It basically just nudges it up a bit, when I re-did the component I tried to get it to match exactly what was there before. Here's a SS with & without this style (lemme know if you prefer to remove entirely):
<img width="494" alt="Screen Shot 2021-03-03 at 12 05 26 AM" src="https://user-images.githubusercontent.com/12075600/109774156-e7e7f400-7bb4-11eb-9106-102792998cf1.png">

Reply:
Got it - I think it's fine without nudging it up. As long as everything is vertically centered in that area, we should be good.